backend/src/main/java/net/vidux/camhub/discovery/CameraScanViduxHelper.java

```diff
@@ -0,0 +1,34 @@
+package net.vidux.camhub.discovery;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+class CameraScanViduxHelper implements CameraScan {
+
+  @Autowired private ViduxHelperWrapper viduxHelperWrapper;
+
+  @Autowired private RawCameraDataFactory rawCameraDataFactory;
```

Comment:
http://olivergierke.de/2013/11/why-field-injection-is-evil/

Constructor injection-t használjunk!